src/SummaryStats.jl

```diff
@@ -0,0 +1,71 @@
+"""
+    SummaryStats
+
+Supertype for every type of summary statistic. 
+
+"""
+
+abstract type SummaryStats end
```

Comment:
Yes this abstract type looks good

Reply:
Am wondering if `SummaryStats` should be an `AbstractFunction` instead?

Reply:
Didn't know about this, will look into it !